sda-commons-server-morphia/src/main/java/org/sdase/commons/server/morphia/internal/MongoClientBuilder.java

```diff
@@ -78,10 +80,12 @@ public MongoClient build(Environment environment) {
   private MongoClient createMongoClient()
       throws KeyStoreException, CertificateException, NoSuchAlgorithmException, IOException,
           KeyManagementException {
-
     if (configuration.isUseSsl()) {
       mongoClientOptionsBuilder.sslEnabled(true);
-      SSLContext sslContext = createSslContextIfAnyCertificatesAreConfigured();
+      // use sslContext created with env variable by default
+      if (configuration.getCaCertificate() != null && !configuration.getCaCertificate().isEmpty()) {
```

Comment:
NIT:
```suggestion
      if (StringUtils.isNotBlank(configuration.getCaCertificate()) {
```
We could also remove the blank check within the private method `createSslContextIfAnyCertificatesAreConfigured`

Reply:
I'll do that when rebasing to master.